src/deploy/table_contact_policy.sql

```diff
@@ -12,34 +13,52 @@ GRANT INSERT, UPDATE, DELETE ON TABLE maevsi.contact TO maevsi_account;
 
 ALTER TABLE maevsi.contact ENABLE ROW LEVEL SECURITY;
 
--- Only display contacts referencing the invoker's account.
--- Only display contacts authored by the invoker's account.
--- Only display contacts for which an accessible invitation exists.
+-- Only display contacts referencing the invoker's account, omit contacts authored by a blocked account.
+-- Only display contacts authored by the invoker's account, omit contacts referring to a blocked account.
+-- Only display contacts for which an accessible invitation exists, omit contacts auhored by a blocked account or referring to a blocked account.
```

Comment:
```suggestion
-- Only display contacts for which an accessible invitation exists.
```
The last case is not covered here, it's implicitly covered by the word `accessible` in the current description. We should just describe the code as it is below here. When we were to change something about the way `invitation_contact_ids` works, we would surely not remember to update the description here as well.